projects/samples/contests/robocup/controllers/model_verifier/model_verifier.py

```diff
@@ -0,0 +1,1044 @@
+from controller import Supervisor
```

Comment:
We should add an Apache 2.0 license header here (if you agree to release it under the Apache 2.0 license).

Reply:
That's done